images/haproxy/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN [ ! -f /usr/share/copyrights.tar.gz ] || tar -C / -xzvf /usr/share/copyright
 # - bash (ldd is a bash script and debian-base removes bash)
 # - procps (for `kill` which kind needs)
 RUN apt update && \
-    apt install -y --no-install-recommends haproxy=2.2.\* \
+    apt install -y --no-install-recommends haproxy=2.6.\* \
```

Comment:
I can't remember now why we had to stick on 2.2, I think that there was some behavior change we depended on it

Reply:
just switch back to the prevous debian version, we'll revisit later 
cc: @BenTheElder